app_dart/lib/src/model/appengine/time_series_value.dart

```diff
@@ -64,6 +65,10 @@ class TimeSeriesValue extends Model {
   @JsonKey(name: 'Revision')
   String revision;
 
+  /// The branch of [commit].
+  @StringProperty(propertyName: 'Branch')
+  String branch;
```

Comment:
Is there a plan in place to update past time series data include the branch field? Or will this only work on new timeseries data?

Reply:
I have updated all historical data to have the branch property over the weekend.